### ISSUE TITLE
Endless Mode

### ISSUE THREAD

**Issue by Flyspeck101** [tetrio/issues#987]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

I've always wanted an endless mode for TETR.IO. Not just Zen Mode, but a new solo mode that has its own leaderboard and slot in the player profile/Tetra Channel. Playing continuously in this format is something that has been implemented in many other games, and is a part of the Tetris Guideline. 

### Describe the solution you'd like

 This new gamemode would have the objective of gaining as much score as possible. It should use a similar levelling system as Blitz, with the addition of decreased Lock Delay with higher levels. This gamemode would provide players with a new way to compete with others, especially for those with a large amount of time on their hands who would find that the existing solo gamemodes become boring after a while. Additionally, this gamemode would benefit slower players, as with 40 Lines and Blitz, the main factor is speed, while with this new endless gamemode, slower players would still be able to reach similar results. At the same time, skill would still be needed to reach higher scores, as players would need to play efficiently in order to get higher scores. It would also challenge players as the gravity increases and the Lock Delay decreases, since it would require fast reaction times to avoid misdropping. 

### Describe alternatives you've considered

This gamemode is easily made through the Custom Game mode, however there is no leaderboard for this. Additionally, players could just go to another Tetris game to play this gamemode.

**Comment by Flyspeck101** [comment]
feat: solo

**Comment by Flyspeck101** [comment]
> its endless in zen


I don't think you read my post at all. Maybe try actually understanding what I'm suggesting before replying with something like this.

**Comment by JadiPlayz** [comment]
> > It's endless in Zen
> 
> I don't think you read my post at all. Maybe try understanding what I'm suggesting before replying with something like this.

oh
